### PR TITLE
Add 1.3.8 release notes

### PR DESCRIPTION
**Pull request by aoguan1990**
Signed-off-by: Aozixuan Priscilla Guan <aoguan@amazon.com>

### Check List
- [ ] All tests pass
  - [ ] `yarn test:jest`
  - [ ] `yarn test:jest_integration`
  - [ ] `yarn test:ftr`
- [ ] New functionality includes testing.
- [ ] New functionality has been documented.
- [ ] Update [CHANGELOG.md](./../CHANGELOG.md)
- [X] Commits are signed per the DCO using --signoff

### CODE REVIEW

File: release-notes/opensearch-dashboards.release-notes-1.3.8.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# Version 1.3.8 Release Notes
+
+
+### 🛡 Security
+
+- [CVE-2022-25901] Bump supertest ([#3326](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3326))
+- [CVE-2022-25860] Bumps simple-git from 3.15.1 to 3.16.0 ([#3345](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3345))
```
Comment (ananzh) [inline release-notes/opensearch-dashboards.release-notes-1.3.8.md:7]: Can we update all `Bumps` to `Bump`? It is better to modify a bit to make it consistent with other release note, ex https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/release-notes/opensearch-dashboards.release-notes-2.5.0.md

File: release-notes/opensearch-dashboards.release-notes-1.3.8.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+
+### 📝 Documentation
+
+- Fix documentation link for date math ([#3207](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3207))
+
+
+### 🔩 Tests
+
+- Prevent `AbortError` from breaking unit tests for ExecutionContract ([#3299](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3299))
```
Comment (ananzh) [inline release-notes/opensearch-dashboards.release-notes-1.3.8.md:32]: Another 3299

File: release-notes/opensearch-dashboards.release-notes-1.3.8.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+### 📈 Features/Enhancements
+
+- [Optimizer] Fixes misleading embeddable plugin error message ([#3043](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3043))
+- [Optimizer] Trim trailing slashes before checking no-restricted-path rule ([#3020](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3020))
+
+
+### 🚞 Infrastructure
+
+- Lock workflow tests to Chrome and ChromeDriver 107 as the last combination that run on Node.js v10 ([#3299](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3299))
```
Comment (ananzh) [inline release-notes/opensearch-dashboards.release-notes-1.3.8.md:20]: Is #3299 titled `Enhance the Build and Test workflow `? I see #3299 in both Infrastructure and Tests... based on the contents, maybe putting it in one place and titling it as its original tile will cause less confusion?

File: release-notes/opensearch-dashboards.release-notes-1.3.8.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
+# Version 1.3.8 Release Notes
+
```
Comment (joshuarrrr) [inline release-notes/opensearch-dashboards.release-notes-1.3.8.md:2]: nit - there should only be one blank line between sections.

File: release-notes/opensearch-dashboards.release-notes-1.3.8.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+
+### 📈 Features/Enhancements
+
+- [Optimizer] Fixes misleading embeddable plugin error message ([#3043](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3043))
+- [Optimizer] Trim trailing slashes before checking no-restricted-path rule ([#3020](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3020))
+
+
+### 🚞 Infrastructure
+
```
Comment (ananzh) [inline release-notes/opensearch-dashboards.release-notes-1.3.8.md:19]: I think you missed these:
https://github.com/opensearch-project/OpenSearch-Dashboards/pull/2966
https://github.com/opensearch-project/OpenSearch-Dashboards/pull/1190
https://github.com/opensearch-project/OpenSearch-Dashboards/pull/2990
https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3020
https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3009

File: release-notes/opensearch-dashboards.release-notes-1.3.8.md [hunk 6]
```diff
@@ -0,0 +1,32 @@
+# Version 1.3.8 Release Notes
+
+
+### 🛡 Security
+
+- [CVE-2022-25901] Bump supertest ([#3326](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3326))
```
Comment (joshuarrrr) [inline release-notes/opensearch-dashboards.release-notes-1.3.8.md:6]: For consistency, let's add the from/to version numbers

File: release-notes/opensearch-dashboards.release-notes-1.3.8.md [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+- [CVE-2022-25860] Bumps simple-git from 3.15.1 to 3.16.0 ([#3345](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3345))
+- [CVE-2022-46175] Bumps json5 version from 1.0.1 and 2.2.1 to 1.0.2 and 2.2.3 ([#3201](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3201))
+- [CVE-2022-25912] Bumps simple-git from 3.4.0 to 3.15.0 ([#3036](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3036))
+
+
+### 📈 Features/Enhancements
+
+- [Optimizer] Fixes misleading embeddable plugin error message ([#3043](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3043))
+- [Optimizer] Trim trailing slashes before checking no-restricted-path rule ([#3020](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3020))
```
Comment (joshuarrrr) [inline release-notes/opensearch-dashboards.release-notes-1.3.8.md:15]: nit - these seem more like bug fixes than enhancements to me. Which is also better for a patch release, where we shouldn't really be introducing new features.

**Review by joshuarrrr — COMMENTED** [review]
A couple minor suggestions

File: release-notes/opensearch-dashboards.release-notes-1.3.8.md [hunk 8]
```diff
@@ -0,0 +1,32 @@
...
+
+
+### 🚞 Infrastructure
+
+- Lock workflow tests to Chrome and ChromeDriver 107 as the last combination that run on Node.js v10 ([#3299](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3299))
+- Remove github-checks-reporter ([#3126](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3126))
```
Comment (ananzh) [inline release-notes/opensearch-dashboards.release-notes-1.3.8.md:21]: I think we did not backport this to 1.3. I think we should change the label and manual backport for 2.5 and 1.3.9.

@joshuarrrr  what do you think?

**Review by ananzh — CHANGES_REQUESTED** [review]

File: release-notes/opensearch-dashboards.release-notes-1.3.8.md (hunk 7 above)
Comment (ananzh) [reply release-notes/opensearch-dashboards.release-notes-1.3.8.md:15]: +1

File: release-notes/opensearch-dashboards.release-notes-1.3.8.md (hunk 5 above)
Comment (ananzh) [reply release-notes/opensearch-dashboards.release-notes-1.3.8.md:19]: Could you help me to double check and categorize them? Thx

File: release-notes/opensearch-dashboards.release-notes-1.3.8.md (hunk 4 above)
Comment (aoguan1990) [reply release-notes/opensearch-dashboards.release-notes-1.3.8.md:2]: Thanks for the comments! Address it as per suggestion.

File: release-notes/opensearch-dashboards.release-notes-1.3.8.md (hunk 7 above)
Comment (aoguan1990) [reply release-notes/opensearch-dashboards.release-notes-1.3.8.md:15]: Move these two PR to bug fixes session.

File: release-notes/opensearch-dashboards.release-notes-1.3.8.md (hunk 1 above)
Comment (aoguan1990) [reply release-notes/opensearch-dashboards.release-notes-1.3.8.md:7]: Thanks for the comments! Address it as per suggestion.

File: release-notes/opensearch-dashboards.release-notes-1.3.8.md (hunk 6 above)
Comment (aoguan1990) [reply release-notes/opensearch-dashboards.release-notes-1.3.8.md:6]: Change it to ` Bump supertest from 2.0.5 to 2.0.12`

File: release-notes/opensearch-dashboards.release-notes-1.3.8.md (hunk 2 above)
Comment (aoguan1990) [reply release-notes/opensearch-dashboards.release-notes-1.3.8.md:32]: Remove the entry under `Test`.

File: release-notes/opensearch-dashboards.release-notes-1.3.8.md (hunk 3 above)
Comment (aoguan1990) [reply release-notes/opensearch-dashboards.release-notes-1.3.8.md:20]: Remove the entry under Test.

File: release-notes/opensearch-dashboards.release-notes-1.3.8.md (hunk 8 above)
Comment (aoguan1990) [reply release-notes/opensearch-dashboards.release-notes-1.3.8.md:21]: Removed #3126 from 1.3.8 release.

File: release-notes/opensearch-dashboards.release-notes-1.3.8.md (hunk 5 above)
Comment (aoguan1990) [reply release-notes/opensearch-dashboards.release-notes-1.3.8.md:19]: Add the following PRs:
https://github.com/opensearch-project/OpenSearch-Dashboards/pull/2966
https://github.com/opensearch-project/OpenSearch-Dashboards/pull/1190
https://github.com/opensearch-project/OpenSearch-Dashboards/pull/2990
https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3009

Comment (ananzh) [reply release-notes/opensearch-dashboards.release-notes-1.3.8.md:18]: Haha what about #3020?

**Review by ananzh — APPROVED** [review]

**Review by joshuarrrr — APPROVED** [review]

**Pull request merged**